projects/plugins/debug-helper/changelog/add-protect-threat-descriptions

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Since we're adding functionality and not really making a bugfix I'd say this is probably better suited for a minor change instead of a patch one. Otherwise it's looking good and testing fine!